client/api_key.go

```diff
@@ -21,8 +21,8 @@ type ApiKeyCreatePayloadWith struct {
 	OrganizationId string `json:"organizationId"`
 }
 
-func (ac *ApiClient) ApiKeyCreate(payload ApiKeyCreatePayload) (*ApiKey, error) {
-	organizationId, err := ac.organizationId()
+func (client *ApiClient) ApiKeyCreate(payload ApiKeyCreatePayload) (*ApiKey, error) {
```

Comment:
Golang coding convention in general is one letter.
But it might be clearer to name it `client` instead of `c`.